### PR TITLE
Build out v1 of the service

### PR DESCRIPTION
**Pull request by jayzes**
Get events flowing from Kinesis into a DB for querying

### CODE REVIEW

**Comment by jayzes** [comment]
@Justin-Holmes i'm going to try to make this live shortly-ish - would love a quick once-over when you have a chance. Not a lot to it since we're moving to Spark for the streaming calculation.

File: spec/interactors/create_event_from_stream_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    it 'silently drops duplicated records' do
+      2.times do
+        described_class.call(kind: 'post_was_viewed', record:  record)
+      end
+      expect(Impression.count).to eq(1)
+    end
+  end
+
+  describe 'with a record that has no viewer' do
```
Comment (Justin-Holmes) [inline spec/interactors/create_event_from_stream_spec.rb]: Is this case for post impressions from logged out users?

**Comment by Justin-Holmes** [comment]
@jayzes Looks good!

File: spec/interactors/create_event_from_stream_spec.rb (hunk 1 above)
Comment (jayzes) [inline spec/interactors/create_event_from_stream_spec.rb]: Exactly

**Pull request merged**